### PR TITLE
feat: Add `email` field to `GroupMember`

### PR DESCRIPTION
**Pull request by candiduslynx**
This is required for https://github.com/cloudquery/cloudquery/issues/10117.
Although the `email` attribute is only visible to group Owners for any [enterprise user](https://docs.gitlab.com/ee/user/enterprise_user/index.html) (see [limitations](https://docs.gitlab.com/ee/api/members.html#limitations)), it'd be nice to have this included.

Might be safely ditched in favor of #1703 (I presume #1703 is a better one)

### CODE REVIEW

**Comment by candiduslynx** [comment]
@svanharmelen could you please take a look?

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,6 @@ _testmain.go
 *.iml
 *.swp
 *.swo
+
+# vendor
+vendor
```
Comment (svanharmelen) [inline .gitignore:33]: Please revert this change...

**Review by svanharmelen — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (candiduslynx) [reply .gitignore:33]: Any reason why this should be left included?

**Comment by candiduslynx** [comment]
@svanharmelen I've reverted vendor change + added the typo fix in here

File: .gitignore (hunk 1 above)
Comment (svanharmelen) [reply .gitignore]: Because this package doesn't have a `vendor` directory checked into git, so excluding it is actually no-op 😉 If you do anything with a vendor directory in your code, then it makes much more sense to exclude the directory in your own gitignore file.

**Review by svanharmelen — APPROVED** [review]
Thanks 👍🏻

File: .gitignore (hunk 1 above)
Comment (candiduslynx) [reply .gitignore]: go mod vendor -> filled in the vendor dir -> need to actually check what files you want to add instead of all...

**Comment by candiduslynx** [comment]
I'd still prefer to go through with #1703, though...

**Comment by candiduslynx** [comment]
@svanharmelen any ETA on the release that will include this fix?

**Comment by svanharmelen** [comment]
Maybe first add a PR for the vendor stuff? Or are you good without that?

**Comment by candiduslynx** [comment]
Yep, in a minute!

**Comment by candiduslynx** [comment]
@svanharmelen #1704 is there.

**Comment by svanharmelen** [comment]
Cool, noticed on minor issue with the additional. Will merge and release when that is fixed/updated 👍🏻

**Comment by candiduslynx** [comment]
updated

**Comment by svanharmelen** [comment]
Released `v0.83.0`
